build.gradle

```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.4.10'
+    ext.kotlin_version = '1.5.32'
```

Comment:
Same here. This helped the tests pass. Only the tests use Kotlin though so it shouldn't affect our builds.